inc/Game.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <string>
+#include "../inc/Frame.hpp"
+
+class Game
+{
+    std::string name_;
+    Frame frame[12];
+public:
+    std::string getName() const;
+    void setName(const std::string &);
+    int score() const;
+    int checkValue(const std::string & value);
```

Comment:
In TDD you should not write any code that is not used. So checkValue, checkIsStrikeOrSpare and setValueFrameAndPrevFrames are not needed. YAGNI.